05/Dog.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "Animal.h"
+
+namespace samples
+{
+	class Dog : public Animal
+	{
+	public:
+		Dog(int age, char* address);
+
+		const char* GetAddress() const;
```

Comment:
static

Reply:
So you want me to make this function return mHomeAddress as a static variable?